client/app/pages/queries/query.html

```diff
@@ -74,6 +74,22 @@
             </a>
         </span>
       </h3>
+      <div class="compare-query-version">
+        <span class="zmdi zmdi-archive"></span>
+        <span class="text-muted">Version</span>
+        <strong>{{query.version}}</strong>
+        &gt;
+        <span>Compare to</span>
+        <select id="version-choice">
+          <option ng-repeat="version in versions" value="{{version.object_version}}">
+            {{version.object_version}}
+          </option>
+        </select>
+        <button class="btn btn-s btn-default" ng-click="compareQueryVersion()">
+          <span class="zmdi zmdi-play"></span>
+          Compare
+        </button>
+      </div>
```

Comment:
Yea good call on more screenshots I was a bit too focused on the modal. The dropbox example wasn't the way we'd intended to do it though I can see some of its merits - we'd gone for a more general "compare current to..." approach. 

This change would require a more significant time investment so we might need to see if it fits Mozilla's needs. Tagging @rafrombrc on this.